scripts/openmc-plot-mesh-tally

```diff
@@ -0,0 +1,325 @@
+        count = 0
+        for f in selectedTally.filters:
+            filterType = f.short_name
+            if filterType == 'Mesh':
+                continue
+            count += 1
+
+            # Create label and combobox for this filter
+            label = tk.Label(self.selectFrame, text=self.labels[filterType])
+            label.grid(row=count+6, column=0, sticky=tk.W)
+            combobox = ttk.Combobox(self.selectFrame, state='readonly')
+            self.filterBoxes[filterType] = combobox
+
+            # Set combobox items
+            if filterType in ['Energy', 'Energyout']:
+                combobox['values'] = ['{0} to {1}'.format(*f.bins[i:i+2])
+                                      for i in range(len(f.bins) - 1)]
+            else:
+                combobox['values'] = [str(i) for i in f.bins]
+
+            combobox.current(0)
+            combobox.grid(row=count+6, column=1, sticky=tk.W+tk.E)
+            combobox.bind('<<ComboboxSelected>>', self.redraw)
+
+        # If There are no filters, leave a 'None available' message
+        if count == 0:
+            count += 1
+            label = tk.Label(self.selectFrame, text="None Available")
+            label.grid(row=count+6, column=0, sticky=tk.W)
+
+        self.redraw()
+
+    def redraw(self, event=None):
+        basis = self.basisBox.current() + 1
+        axial_level = self.axialBox.current() + 1
+        mbvalue = self.meanBox.get()
+
+        # Get selected tally
+        tally_id = self.meshTallies[self.tallyBox.current()]
+        selectedTally = self.datafile.tallies[tally_id]
+
+        # Create spec_list
+        spec_list = []
+        for f in selectedTally.filters:
+            if f.short_name == 'Mesh':
+                mesh_filter = f
+                continue
+            elif f.short_name in ['Energy', 'Energyout']:
+                index = self.filterBoxes[f.short_name].current()
+                ebin = (f.bins[index], f.bins[index + 1])
+                spec_list.append((type(f), (ebin,)))
+            else:
+                index = self.filterBoxes[f.short_name].current()
+                spec_list.append((type(f), (index,)))
+
+        dims = (self.nx, self.ny, self.nz)
+
+        text = self.basisBox.get()
+        if text == 'xy':
+            h_ind = 0
+            v_ind = 1
+        elif text == 'yz':
+            h_ind = 1
+            v_ind = 2
+        else:
+            h_ind = 0
+            v_ind = 2
+
+        axial_ind = 3 - (h_ind + v_ind)
+        dims = (dims[h_ind], dims[v_ind])
+
+        mesh_dim = len(self.mesh.dimension)
+        if mesh_dim == 3:
+            mesh_indices = [0,0,0]
+        else:
+            mesh_indices = [0,0]
+
+        matrix = np.zeros(dims)
+        for i in range(dims[0]):
+            for j in range(dims[1]):
+                if mesh_dim == 3:
+                    mesh_indices[h_ind] = i + 1
+                    mesh_indices[v_ind] = j + 1
```

Comment:
Optional: I would just combine these two lines.  It seems more readable this way to me.

```
nucname = nuclide.attrib['name'].replace('-', '')
```